tutorials/openmm-torch-nnpops.ipynb

```diff
@@ -0,0 +1,440 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "oRr7FSA13_Wv"
+      },
+      "source": [
+        "# Tutorial: a simple simulation of alanine dipeptide with ANI-2x using OpenMM-Torch and NNPOps\n",
+        "\n",
+        "You can run this tutorial directly in your browser: [![Open On Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/raimis/openmm-torch/blob/example/tutorials/openmm-torch-nnpops.ipynb)\n",
+        "\n",
+        "Covered topics:\n",
+        " * Installation of the software with [Conda](https://docs.conda.io/)\n",
+        " * Creation of an NNP (neural network potential) with [TorchANI](https://aiqm.github.io/torchani/)\n",
+        " * Acceleration of [ANI-2x](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/60c747e9567dfec574ec48df/original/extending-the-applicability-of-the-ani-deep-learning-molecular-potential-to-sulfur-and-halogens.pdf) with [NNPOps](https://github.com/openmm/NNPOps) \n",
+        " * Integration of [OpenMM](https://openmm.org/) and [PyTorch](https://pytorch.org/) and with [OpenMM-Torch](https://github.com/openmm/openmm-torch)\n",
+        " * Setup and execution of a simulation"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "BKG1ZNq94f-Z"
+      },
+      "source": [
+        "## Install Conda\n",
+        "\n",
+        "[Conda](https://docs.conda.io/) is a package and environment manager. On Google Colab, Conda is installed with [conda-colab](https://github.com/jaimergp/condacolab). On your computer, you should follow these [installation instructions](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).\n",
+        "\n",
+        "⚠️ Do not use conda-colab on your computer!"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "EcB2lc1s3ZXG",
+        "outputId": "1ce0bad8-a050-48d5-c18a-9e49fbeb11c2"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "⏬ Downloading https://repo.anaconda.com/miniconda/Miniconda3-py37_4.9.2-Linux-x86_64.sh...\n",
+            "📦 Installing...\n",
+            "📌 Adjusting configuration...\n",
+            "🩹 Patching environment...\n",
+            "⏲ Done in 0:00:21\n",
+            "🔁 Restarting kernel...\n"
+          ]
+        }
+      ],
+      "source": [
+        "!pip install -q condacolab\n",
+        "import condacolab\n",
+        "condacolab.install_miniconda()"
```

Comment:
What about using just `condacolab.install()`, which will install `mambaforge` (miniforge + mamba) instead of `miniconda`? This can speed up installs.

Reply:
I tried `mamba`. It works on a local machine, but fail on Colab due to a pinning issue.